api/helpers/connection.go

```diff
@@ -0,0 +1,35 @@
+package helpers
```

Comment:
we were thinking of pushing all the connection related service code inside service folder